src/domain/reputation.ts

```diff
@@ -0,0 +1,35 @@
+import { Address, store } from '@graphprotocol/graph-ts';
+import { Reputation } from '../types/NativeReputation/Reputation';
+import { Rep } from '../types/schema';
+
+export function getReputation(id: string): Rep {
+  let reputation = store.get('Rep', id) as Rep;
+  if (reputation == null) {
+    reputation = new Rep();
```

Comment:
does this line is needed here ?

Reply:
does store.get('Rep', id) as Rep; does not to the job ?

Reply:
could be null